src/snovault/schema_utils.py

```diff
@@ -174,11 +213,12 @@ def linkFrom(validator, linkFrom, instance, schema):
         if validator._serialize:
```

Comment:
I don't fully understand what this is doing, but I didn't understand before you modified it either.

Reply:
I think this is just a flag for whether the validator is supposed to store the serialized, validated value in validator._validated or not. I don't know why that's not desired in some cases, but we're just conforming to the internals of the `jsonschema` library.